pkg/controller/provider/container/ovirt/client.go

```diff
@@ -226,3 +234,365 @@ func GetInsecureSkipVerifyFlag(secret *core.Secret) bool {
 
 	return insecureSkipVerify
 }
+
+func DownloadOVFTestConection(secret *core.Secret) (ok bool, err error) {
+
+	var diskIdOVF string
+
+	url := fmt.Sprint(string(secret.Data["url"]), "/ovirt-engine/api")
+
+	// Create the connection to the server:
+	conn, err := ovirtsdk.NewConnectionBuilder().
+		URL(url).
+		Username(string(secret.Data["username"])).
+		Password(string(secret.Data["password"])).
+		Insecure(true).
+		Compress(true).
+		Timeout(time.Second * 10).
+		Build()
+	if err != nil {
+		err = liberr.Wrap(err)
+		return
+	}
+	defer conn.Close()
+
+	//Find OVF disk ID
+	disksService := conn.SystemService().DisksService()
+	diskResponse, err := disksService.List().Send()
+	if err != nil {
+		err = liberr.Wrap(err)
+		return
+	}
+
+	disks, ok := diskResponse.Disks()
+
+	if ok {
+		//get OVF disk ID
+		for _, disk := range disks.Slice() {
+			if diskName, ok := disk.Name(); ok {
+				if diskName == "OVF_STORE" {
+					if diskId, ok := disk.Id(); ok {
+						diskIdOVF = diskId
+					}
+					break
+				}
+			}
+		}
+	}
+
+	// Get a disk identified by uuid
+	diskService := disksService.DiskService(diskIdOVF)
+	diskRequest := diskService.Get()
+	diskRequestSent, err := diskRequest.Send()
+	if err != nil {
+		err = liberr.Wrap(err)
+		return
+	}
+	disk, ok := diskRequestSent.Disk()
```

Comment:
do we have to call get-disk here or can we use the disk we got from the 'list' call above?

Reply:
you're right, no need removed the entire section.